packages/x-scheduler/src/joy/internals/components/time-grid/TimeGrid.tsx

```diff
@@ -0,0 +1,161 @@
+'use client';
+import * as React from 'react';
+import clsx from 'clsx';
+import { SchedulerValidDate } from '@mui/x-scheduler/primitives/utils/adapter/types';
```

Comment:
Nitpick: One lone absolute import. 🙈 
```suggestion
import { SchedulerValidDate } from '../../../../primitives/utils/adapter/types';
```

Reply:
True! After this [change](https://github.com/mui/mui-x/pull/18296#discussion_r2139741956) I believe using `import { SchedulerValidDate } from '@mui/x-scheduler/primitives/models';` is the right approach now, correct?

Reply:
Thanks for clarifying! That pattern wasn’t obvious to me 😆 Now it makes sense!